AndroidTest/app/src/main/java/cloudant/com/androidtest/keyprovider/KeyStorageTests.java

```diff
@@ -0,0 +1,104 @@
+package cloudant.com.androidtest.keyprovider;
+
+import android.test.AndroidTestCase;
+
+import com.cloudant.sync.datastore.encryption.KeyData;
+import com.cloudant.sync.datastore.encryption.KeyStorage;
+
+import java.util.Arrays;
+
+public class KeyStorageTests extends AndroidTestCase {
+
+    // Green Path Tests
+    public void testKeyDataSaveAndRetreive() {
```

Comment:
Changes requested in comment included in https://github.com/cloudant/sync-android/commit/dd276d263bfc9c5ddebb047cecd27230cb074f63